### ISSUE TITLE
Approve/unapprove threads moderator permission broken

### ISSUE THREAD

**Issue by Matslom** [mybb/mybb#2592]
When adding a moderator, there are two options:

Can approve/unapprove threads?

Can approve/unapprove posts?

However, enabling the setting to (un)approve threads does not show the option to (un)approve threads on the forum display, instead it checks the posts setting:

PHP Code:if(is_moderator(&#36;fid,&nbsp;"canapproveunapproveposts")){&nbsp;&nbsp;&nbsp;&nbsp;eval("&#092;&#36;inlinemodapproveunapprove&nbsp;=&nbsp;&#092;"".&#36;templates-&gt;get("forumdisplay_inlinemoderation_approveunapprove")."&#092;";");}&nbsp;
So basically, you can only (un)approve threads, if you have the permission to (un)approve posts. The check in forumdisplay.php should check canapproveunapprovethreads instead. The check in showthread.php seems fine.

Original thread: [Approve/unapprove threads moderator permission broken](https://community.mybb.com/thread-206957.html)

**Comment by Cu8eeeR** [comment]
@Ben-Conway - can you please add "fixed" tag?

**Comment by thelovelyone** [comment]
This seems broken to me after all.

Basically, added a new moderator for a specific forum (not global), allowing all the moderator permissions, however these options:

(un)approve threads & posts
permanent delete

are not available for that moderator.

**Comment by Ben-MyBB** [comment]
Can anyone confirm? @mybb/developers @mybb/sqa
